### PR TITLE
feat: AWS Bedrock example app

### LINKED ISSUE #153: Create AWS Bedrock example application
## Is your feature request related to a problem? Please describe.

Create an AWS Bedrock example application that leverages a webframework and uses v3 of `@aws-sdk/client-bedrock-runtime`.  It should do the following:

 * create endpoint that creates chat completions via promise with the following models(InvokeModelCommand):
   * amazon titan text
   * anthropic claude
   * AI21 labs jurassic 2
   * Cohere
 * create endpoint to create chat completions via stream(InvokeModelWithResponseStreamCommand)
 * create endpoint that creates chat completion, stores relevant ids from `getLlmMesagesIds`  and in a separate API call it would record a feedback event with those ids
 * create endpoint that creates an embedding(InvokeModelCommand with modelId of `amazon.titan-embed`).

**Note**: It should require a user to provide agent api key, application name, api endpoint as well as AWS access secret and key.

### PR DESCRIPTION
**Pull request by svetlanabrennan**
## Links
Fixes #153 

## Details
This is an example AWS bedrock application. It includes a README with steps on how to use this application. It uses the default model configuration [inference parameters](https://docs.aws.amazon.com/bedrock/latest/userguide/model-parameters.html). 

Note: `InvokeModelWithResponseStreamCommand` does not support the A121 model. 

Example outputs from curl requests:

```shell
curl -XPOST -H 'Content-Type: application/json' http://localhost:3000/chat-completion -d '{"message":"How much wood could a woodchuck chuck if a woodchuck could chuck wood?", "model": "amazon-titan"}'

{"requestId":"3afd641e-c930-41abc-a123-c92d123456789","outputText":"\nThe amount of wood a woodchuck could chuck if a woodchuck could chuck wood is a theoretical question that has no known answer. It is a play on words that combines the name of a small animal, the woodchuck, with the phrase \"chuck wood,\" which is not a real action or ability."}% 
```       

```shell
curl -XPOST -H 'Content-Type: application/json' http://localhost:3000/chat-completion-stream -d '{"message":"Explain the rules of jai alai"}'

requestId": a508f66d-b2c7-47f3-8abc-ad688d7abc123456{"outputText":"\nJai alai is a sport that is played with a ball and a wicker basket called a cesta. The game is played on a court that ","index":0,"totalOutputTextTokenCount":null,"completionReason":null,"inputTextTokenCount":9}{"outputText":"is 131 by 66 feet and has four walls. The court is divided into two halves by a net.\n\nThe game is played by two teams of three players each. The players on each team wear gloves and use a small, hard ball to hit the cesta. The ball must be hit over the net and into the other team's half of the court without touching the walls or the floor.\n\nThe players on each team take turns hitting","index":0,"totalOutputTextTokenCount":129,"completionReason":"LENGTH","inputTextTokenCount":null,"amazon-bedrock-invocationMetrics":{"inputTokenCount":9,"outputTokenCount":129,"invocationLatency":5353,"firstByteLatency":5025}}
```

```shell
curl -XPOST http://localhost:3000/embedding

{"requestId":"cf292d59-125b-4673-adf0-6be47ccec79c","embedding":[0.6015625,0.03125,0.6953125,0.53515625,-0.1796875,0.33984375,-0.12207031,-0.0015945435,-0.4921875,-0.26953125,0.38476562,-0.39453125,0.765625,0.7890625,-1.859375,0.578125,0.21289062,-0.40039062,-0.08203125,0.16992188,-0.14355469,-0.024902344,-0.91015625,0.8359375,0.21289062,-0.50390625,-0.5703125,0.640625,-0.71875,0.2578125,0.6484375,0.890625,0.77734375,0.027832031,0.21875,0.16308594,0.028808594,1.15625,-0.5078125,0.38671875,0.6796875,-0.765625,0.8515625,0.5703125]}% 
```

### CODE REVIEW

**Review by jsumners-nr — APPROVED** [review]
Looks good to me.

File: aws-bedrock-app/newrelic.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+     */
+    level: 'trace'
+  },
+  /**
+   * When true, all request headers except for those listed in attributes.exclude
+   * will be captured for all traces, unless otherwise specified in a destination's
+   * attributes include/exclude lists.
+   */
+  allow_all_headers: true,
+  ai_monitoring: {
```
[the remark this answers is not in the document]
Comment (svetlanabrennan) [reply aws-bedrock-app/newrelic.js:24]: Done

File: aws-bedrock-app/index.js [hunk 2]
```diff
@@ -0,0 +1,201 @@
+'use strict'
+require('dotenv').config();
```
[the remark this answers is not in the document]
Comment (svetlanabrennan) [reply aws-bedrock-app/index.js]: Done

File: aws-bedrock-app/package.json [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+  "scripts": {
+    "start": "node --env-file .env -r newrelic index.js",
+    "start:debug": "node --inspect-brk --env-file .env -r newrelic index.js",
+    "start:no-agent": "node --env-file .env index.js"
+  },
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "@aws-sdk/client-bedrock-runtime": "^3.484.0",
+    "dotenv": "^16.3.1",
```
[the remark this answers is not in the document]
Comment (svetlanabrennan) [reply aws-bedrock-app/package.json]: Done

**Pull request merged**